charms/istio-pilot/test/unit/test_charm.py

```diff
@@ -76,3 +76,35 @@ def test_main_service_mesh(harness):
             break
     else:
         assert False
+
+
+# def test_main_istio_pilot(harness):
+#     harness.set_leader(True)
+#     harness.add_oci_resource(
+#         "oci-image",
+#         {
+#             "registrypath": "docker.io/istio/pilot:1.5.0",
+#             "username": "",
+#             "password": "",
+#         },
+#     )
+#     harness.begin_with_initial_hooks()
+#     rel_id = harness.add_relation("istio-pilot", "app")
+#     # harness.add_relation_unit(rel_id, "app/0")
+#
+#     relation = harness.model.get_relation("istio-pilot", rel_id)
+#     # data = {
+#     #     "service-name": "service",
+#     #     "service-port": "666",
+#     # }
+#     relation.data[relation.app]._is_mutable = lambda: True
+#     relation.data[relation.app]["_supported_versions"] = "- v1"
+#     # relation.data[relation.app]["data"] = yaml.dump(data)
+#     # harness.update_relation_data(
+#     #     rel_id,
+#     #     "app",
+#     #     {"_supported_versions": "- v1", "data": yaml.dump(data)},
+#     # )
+#     harness.charm.on.istio_pilot_relation_joined.emit(relation)
+#
+#     assert relation.data[harness.charm.app] == None
```

Comment:
Do we want to keep this in here?  I don't have an opinion, just making sure it isn't here by mistake